src/platform/glfw/window.ts

```diff
@@ -847,7 +869,25 @@ export class WindowGlfw extends DwmWindow {
       }
     }
   }
-
+  setCustomCursor(image: Uint8Array, hotspot: Size, position: Position) {
```

Comment:
once the new byte type gets released lets create the struct that way for now this works